dio/lib/src/transformers/utf8_json_transformer.dart

```diff
@@ -0,0 +1,184 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:dio/dio.dart';
+import 'package:dio/src/compute/compute.dart';
+
+/// A [Transformer] that has a fast path for decoding utf8-encoded JSON.
+/// If the response is utf8-encoded JSON and no custom decoder for a Request is specified, this transformer
```

Comment:
What is the `Request` referencing here?

Reply:
the RequestOptions. If a custom decoder is set in to RequestOptions, we respect that, but in this case we cannot use the fused decoder. 
I changed the comments to clarify that.